server/resource_server/service/service_test.go

```diff
@@ -4,15 +4,47 @@ import (
 	"fmt"
 	serverModels "globe-and-citizen/layer8/server/models"
 	"globe-and-citizen/layer8/server/resource_server/dto"
+	"globe-and-citizen/layer8/server/resource_server/emails/verification"
 	"globe-and-citizen/layer8/server/resource_server/models"
 	"globe-and-citizen/layer8/server/resource_server/service"
+	"globe-and-citizen/layer8/server/resource_server/utils/mocks"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 )
 
+const userId = 1
+const adminEmail = "admin@email.com"
+const username = "user"
+const userEmail = "user@email.com"
+const verificationCode = "123456"
+
+var timestamp = time.Date(2024, time.May, 24, 14, 0, 0, 0, time.UTC)
+var timestampPlusTwoSeconds = timestamp.Add(2 * time.Second)
+var now = func() time.Time {
+	return timestamp
+}
+var mockCodeGenerator = &mocks.MockCodeGenerator{
+	VerificationCode: verificationCode,
+}
+var defaultMockSenderService = &mocks.MockEmailSenderService{
+	SendEmailFunc: func(email *models.Email) error {
+		return nil
+	},
+}
+
 type mockRepository struct {
```

Comment:
Could this go in `utils/mocks` with the `MockeEmailSenderService`?

Reply:
It should go to the mocks package. That can be done in another PR, and also there are other mock implementations which should be extracted into a dedicated package to be reused between different test files.